percona-packaging/scripts/pg_percona_telemetry_builder.sh

```diff
@@ -274,7 +274,12 @@ install_deps() {
                 apt-get update
             fi
         fi
-
+        if [[ "x${OS_NAME}" == "xbullseye" ]]; then
```

Comment:
Why this `x` prefix is needed?

Reply:
It isn't needed and should work as intended without the prefix, which I've removed now. This part was copied from another script, hence the prefix got added.